styleguide/_themes/global/scss/grid.scss

```diff
@@ -1,10 +1,20 @@
+$max-row-width: 960px;
+
+@mixin rs-make-row($gutter: $grid-gutter-width) {
+  @include make-row($gutter);
+  margin: 0 auto;
+  max-width: $max-row-width;
+}
```

Comment:
well in theory we're just shifting the 960 restriction so it *should* work.  Should is the key word.  I think this is gonna need a heavy hand as far as QE goes, and probably is gonna extend into next sprint at this point.